src/dispatcher.js

```diff
@@ -130,6 +130,7 @@ class CommandDispatcher {
 				if(cmdMsg.command) {
 					if(!cmdMsg.command.isEnabledIn(message.guild)) {
 						if(!cmdMsg.command.unknown) {
+							this.client.emit('disabledCommandRun');
```

Comment:
You should also include the source message in the event.

Reply:
Thanks for letting me know. It's been added